### PR TITLE
Addressing Server Admin review comments

### LINKED ISSUE #24643: Address comments on the Server Adminstration guide
### Description

Miscellaneous comments

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by andymunro**
Closes #24643

@ahus1 @vaceksimon Can you verify this.   This addressed most issues you raised.

### CODE REVIEW

**Review by ahus1 — DISMISSED** [review]
@andymunro - your PR references a GitHub issue which is not related to the PR. GitHub PRs can only reference GitHub issues, not internal JIRA issues. Please duplicate the JIRA issue to a public GitHub issue first so it is clear what you intent to change. 

UPDATE: the commit in this PR doesn't contain a signoff, please add this as well. See https://www.keycloak.org/2023/10/dco for the details.

File: docs/documentation/server_admin/topics/authentication/flows.adoc [hunk 1]
```diff
@@ -106,24 +106,23 @@ Three factors determine the behavior of flows and sub-flows.
...
+You can add sub-flows to top-level flows with the *Add sub-flow* button. The *Add sub-flow* button displays the *Create Execution Flow* page. This page is similar to the *Create Top Level Form* page. The difference is that the *Flow Type* can be *basic* (default) or *form*. The *form* type constructs a sub-flow that generates a form for the user, similar to the built-in *Registration* flow.
 Sub-flows success depends on how their executions evaluate, including their contained sub-flows. See the <<_execution-requirements, execution requirements section>> for an in-depth explanation of how sub-flows work.
 
 [NOTE]
 ====
 After adding an execution, check the requirement has the correct value.
 ====
 
-All elements in a flow have a *Delete* option in the *Actions* menu. This action removes the element from the flow.
-Executions have a *⚙️* menu item (the gear icon) to configure the execution. It is also possible to add executions and sub-flows to sub-flows with the *Add step* and *Add flow* links.
```
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/authentication/flows.adoc:126]: I apologize for the incorrect comment I left on the gear icon on the Jira ticket. The gear menu item to configure the execution is still there, it's just not there for every single execution.

File: docs/documentation/server_admin/topics/authentication/flows.adoc [hunk 2]
```diff
@@ -262,7 +261,6 @@ Now you configure the flow for the first authentication level.
 . Select *Conditional - Level Of Authentication* from the list.
 . Click *Add*.
 . Click *Required* for the *Conditional - Level Of Authentication* authentication type to set its requirement to required.
-. Click *+* menu of the *Conditional - Level Of Authentication*.
 . Click *⚙️* (gear icon).
```
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/authentication/flows.adoc:264]:
```suggestion
. Click *⚙️* (gear icon) menu of the *Conditional - Level Of Authentication*
```

File: docs/documentation/server_admin/topics/admin-cli.adoc [hunk 3]
```diff
@@ -1933,7 +1933,7 @@ $ kcadm create "authentication/executions/a3147129-c402-4760-86d9-3f2345e401c7/c
 For example:
 [options="nowrap"]
 ----
-$ kcadm get "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r examplerealm
+$ kcadm get "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r demorealm
```
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/admin-cli.adoc:1936]:
```suggestion
$ kcadm.sh get "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r demorealm
```

File: docs/documentation/server_admin/topics/admin-cli.adoc [hunk 4]
```diff
@@ -1948,7 +1948,7 @@ $ kcadm get "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r exam
...
+$ kcadm update "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r demorealm -b '{"id":"dd91611a-d25c-421a-87e2-227c18421833","alias":"my_otp_config","config":{"x509-cert-auth.extendedkeyusage":"","x509-cert-auth.mapper-selection.user-attribute-name":"usercertificate","x509-cert-auth.ocsp-responder-uri":"","x509-cert-auth.regular-expression":"(.*?)(?:$)","x509-cert-auth.crl-checking-enabled":"true","x509-cert-auth.confirmation-page-disallowed":"","x509-cert-auth.keyusage":"","x509-cert-auth.mapper-selection":"Custom Attribute Mapper","x509-cert-auth.crl-relative-path":"crl.pem","x509-cert-auth.crldp-checking-enabled":"false","x509-cert-auth.mapping-source-selection":"Match SubjectDN using regular expression","x509-cert-auth.ocsp-checking-enabled":""}}'
```
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/admin-cli.adoc:1951]:
```suggestion
$ kcadm.sh update "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r demorealm -b '{"id":"dd91611a-d25c-421a-87e2-227c18421833","alias":"my_otp_config","config":{"x509-cert-auth.extendedkeyusage":"","x509-cert-auth.mapper-selection.user-attribute-name":"usercertificate","x509-cert-auth.ocsp-responder-uri":"","x509-cert-auth.regular-expression":"(.*?)(?:$)","x509-cert-auth.crl-checking-enabled":"true","x509-cert-auth.confirmation-page-disallowed":"","x509-cert-auth.keyusage":"","x509-cert-auth.mapper-selection":"Custom Attribute Mapper","x509-cert-auth.crl-relative-path":"crl.pem","x509-cert-auth.crldp-checking-enabled":"false","x509-cert-auth.mapping-source-selection":"Match SubjectDN using regular expression","x509-cert-auth.ocsp-checking-enabled":""}}'
```

File: docs/documentation/server_admin/topics/admin-cli.adoc [hunk 5]
```diff
@@ -1963,5 +1963,5 @@ $ kcadm update "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r e
 For example:
 [options="nowrap"]
 ----
-$ kcadm delete "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r examplerealm
+$ kcadm delete "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r demorealm
```
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/admin-cli.adoc:1966]:
```suggestion
$ kcadm.sh delete "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r demorealm
```

File: docs/documentation/server_admin/topics/authentication/otp-policies.adoc [hunk 6]
```diff
@@ -1,9 +1,15 @@
...
 === One Time Password (OTP) policies
 
-{project_name} has several policies for setting up a FreeOTP or Google Authenticator One-Time Password generator. Click the *Authentication* menu and click the *OTP Policy* tab.
+{project_name} has several policies for setting up a FreeOTP or Google Authenticator One-Time Password generator.
 
-.Otp policy
+.Procedure
+
+. Click *Authentication* in the menu.
+. Click the *Policy* tab.
```
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/authentication/otp-policies.adoc:9]:
```suggestion
. Click the *Policies* tab.
```

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc [hunk 7]
```diff
@@ -77,7 +76,6 @@ The user can choose between using WebAuthn and OTP for the second factor:
 . On the *Conditional 2FA* row, click the plus sign + and select *Add step*.
 . Click *Add step*.
```
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/authentication/webauthn.adoc:77]:
```suggestion
```

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc [hunk 8]
```diff
@@ -77,7 +76,6 @@ The user can choose between using WebAuthn and OTP for the second factor:
...
 . Select *OTP Form* from the list.
-. Click *Add*.
```
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/authentication/webauthn.adoc:80]: This step is perfectly fine and should stay. Step 2 on line 77 is supposed to be the one deleted

File: docs/documentation/server_admin/topics/users/proc-requiring-tac-agreement-at-registration.adoc [hunk 9]
```diff
@@ -16,7 +16,8 @@ image:images/registration-form-with-required-tac.png[]
 * Terms and conditions required action is enabled.
 
 .Procedure
-. Click the *Flows* tab.
+. Click *Authentication* in the menu.
+Click the *Flows* tab.
```
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/users/proc-requiring-tac-agreement-at-registration.adoc:20]:
```suggestion
. Click the *Flows* tab.
```

File: docs/documentation/server_admin/images/user-profile-select-options-simple-i18n.png
Comment (vaceksimon) [inline docs/documentation/server_admin/images/user-profile-select-options-simple-i18n.png:1]: The picture from an older version of Keycloak I had in mind was [user-profile-select-options-custom-validator.png](https://github.com/keycloak/keycloak/blob/main/docs/documentation/server_admin/images/user-profile-select-options-custom-validator.png). This picture is fine as it is.

**Review by vaceksimon — CHANGES_REQUESTED** [review]
After these corrections it's ready for merge. Good job Andy

**Comment by andymunro** [comment]
@ahus1 @vaceksimon Comments addressed and signed off to correct Jira.

**Review by ahus1 — DISMISSED** [review]
DCO looks good to me. Waiting for review of @vaceksimon

**Review by vaceksimon — APPROVED** [review]
LGTM! Ready to merge @ahus1

**Comment by andymunro** [comment]
@ahus1 I can fix one more issue with this guide related to the Account Console broken link

**Comment by andymunro** [comment]
@ahus1 I fixed that last broken link issue.  Ready to merge.

**Review by ahus1 — APPROVED** [review]
LGTM

**Pull request merged**